services/api/handlers/users.js

```diff
@@ -0,0 +1,137 @@
+var boom = require('boom');
+
+module.exports = function(request, reply) {
+  if (request.method === 'get') {
```

Comment:
This style of request handling seems pretty strange. Is there a particular reason you choose to do it this way?

(And yes, I totally realize this was how it was done for skittles before... shitty review on my part)